src/main/java/com/github/paweladamski/httpclientmock/condition/UrlEncodedFormCondition.java

```diff
@@ -0,0 +1,131 @@
+package com.github.paweladamski.httpclientmock.condition;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.http.HttpEntityEnclosingRequest;
+import org.apache.http.NameValuePair;
+import org.apache.http.client.utils.URLEncodedUtils;
+import org.hamcrest.Matcher;
+
+import com.github.paweladamski.httpclientmock.Debugger;
+import com.github.paweladamski.httpclientmock.Request;
+import com.github.paweladamski.httpclientmock.matchers.MatchersMap;
+
+/**
+ * Tests the request body for URL-encoded parameters.
+ * @author Michael Angstadt
+ */
+public class UrlEncodedFormCondition implements Condition {
+  private final MatchersMap<String, String> expected = new MatchersMap<>();
+  private boolean enabled = false;
```

Comment:
Do we really need this parameter?
I think it is unnecessary. When the `expected` map is empty method should return true only if request has zero form parameters.

Now it is actually not possible to say that I want to have a request without form parameters.

For URL parameters it already works in a similar way.

Reply:
Oh, I thought it would fail if the request body contained something other than form parameters. But it does not. Fixed in 78451eae763214a9bb05273999d1d4888e9cb554.